### DISCUSSION TITLE
Possibility to use zero-ui https without domain name

### DISCUSSION THREAD

**Discussion by squromiv** [dec0dOS/zero-ui#94] [Q&A] [answered]
I am newbie to Zerotier and network technologies. I rented VPS with static IP address. Used dockerized [ztncui-aio](https://github.com/key-networks/ztncui-aio) controller. It workes well. Devices are connected.

And now I want to try zero-ui. Following the description, it can work with HTTPS using domain name. If I have no possibility to buy domain name, can I use somehow this feature with IP address of VPS?

**Comment by dsm150** [comment, marked as answer]
Use free ddns services like https://www.duckdns.org/

**Reply by squromiv** [reply]
Thanks! I managed to launch zero-ui by this method.
